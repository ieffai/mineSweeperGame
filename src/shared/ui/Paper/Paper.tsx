import BasePaper, { type PaperProps as BasePaperProps } from '@mui/material/Paper';

export type PaperProps = BasePaperProps;
export const Paper = BasePaper;
export default Paper;
