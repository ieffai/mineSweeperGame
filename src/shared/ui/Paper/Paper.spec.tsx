import { render } from '@testing-library/react';
import Paper from './Paper';

describe('Paper', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<Paper />);
    expect(asFragment()).toMatchSnapshot();
  });
});
