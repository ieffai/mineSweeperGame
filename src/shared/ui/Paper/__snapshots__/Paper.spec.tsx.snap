// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Paper > should render successfully 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1gtchvp-MuiPaper-root"
    style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
  />
</DocumentFragment>
`;
